pkgs/by-name/le/leftwm-theme/package.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchFromGitHub, rustPlatform, openssl, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "leftwm-theme";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "leftwm";
+    repo = pname;
+    rev = "v" + version;
+    sha256 = "sha256-JCNpgdV4v0DzRmELAv+UwPAZ7O2V0KI4t8zirhhQshA";
+  };
+
+  cargoSha256 = "sha256-wit1rAOH9tIu8SpyXNk0GNuBo17liBw4e+JjPQj+uA4=";
+
+  checkFlags = [
+    # direct writing /tmp
+    "--skip=models::config::test::test_config_new"
+    # with network access when testing
+    "--skip=operations::update::test::test_update_repos"
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl ];
+  OPENSSL_NO_VENDOR = 1;
```

Comment:
maybe `env.OPENSSL_NO_VENDOR = 1;`

Reply:
Assumed those were different things, seems I assumed wrong! Will be changed